Webhook timeout (#3254)
fishtown-analytics/docs.getdbt.com: website/docs/docs/deploy/webhooks.md

- `job.run.errored` &mdash; Run errored.

dbt Cloud retries sending each event five times. dbt Cloud keeps a log of each webhook delivery for 30 days. Every webhook has its own **Recent Deliveries** section, which lists whether a delivery was successful or failed at a glance. 

> The timeout for a webhook is 10 sec i.e; dbt cloud webhook processor waits for 10 sec for the endpoint to respond before it times out.

Reviewer: ```suggestion

A webhook in dbt Cloud has a timeout of 10 seconds. This means that if the endpoint doesn't respond within 10 seconds, the webhook processor will time out.
```
